main.html

```diff
@@ -57,7 +57,9 @@
 <script type="text/javascript" src="./js/serial_backend.js"></script>
 <script type="text/javascript" src="./js/data_storage.js"></script>
 <script type="text/javascript" src="./js/fc.js"></script>
-<script type="text/javascript" src="./js/msp.js"></script>
+<script type="text/javascript" src="./js/msp/MSP_codes.js"></script>
```

Comment:
Yes I agree, the naming for MSP_codes.js was just to be consistent with how it was previously defined. I will consider this and make a new naming proposal.